api/src/test/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/MaxPool3DTest.kt

```diff
@@ -0,0 +1,133 @@
+package org.jetbrains.kotlinx.dl.api.core.layer
+
+import org.jetbrains.kotlinx.dl.api.core.KGraph
+import org.jetbrains.kotlinx.dl.api.core.layer.pooling.MaxPool3D
+import org.jetbrains.kotlinx.dl.api.core.loss.EPS
+import org.jetbrains.kotlinx.dl.api.core.shape.shapeFromDims
+import org.junit.jupiter.api.Assertions.assertArrayEquals
+import org.junit.jupiter.api.Test
+import org.tensorflow.EagerSession
+import org.tensorflow.Graph
+import org.tensorflow.op.Ops
+import org.tensorflow.Shape
+import org.junit.jupiter.api.Assertions.assertEquals
+
+internal class MaxPool3DTest {
+
+    private val input = Array(1) {
+        Array(
+            30
+        ) {
+            Array(30) {
+                Array(30) {
+                    FloatArray(3, { 0f })
```

Comment:
Hmm, I agree that it works with zeroes, but could we use other numbers instead or add yet one numerical test for that, what do you think?

Reply:
Sure i will add some numeric tests, working on test data now will commit it soon